corehq/apps/app_manager/static/app_manager/js/app_manager.js

```diff
@@ -555,7 +555,7 @@ hqDefine('app_manager/js/app_manager', [
         });
 
         $('#define-case-type-modal').on('show.bs.modal', function () {
-            var $caseType = $('#new_case_type');
+            var $caseType = $('#new-case-type');
```

Comment:
should this be `#new-case-type-dropdown`, as that was the id changed below?

Reply:
b1e87f09b709714a66f65b8da14169796cf34be1 Thank you for catching it!